lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -669,6 +669,9 @@ function dosomething_campaign_node_view($node, $view_mode, $langcode) {
     // Add signup_data_form variables if needed.
     dosomething_campaign_add_signup_data_form_vars($node);
 
+    // Flag if we need to register an organ donor on the page.
+    dosomething_campaign_add_oragan_donation_vars($node);
```

Comment:
I feel confused by this function call and the name.

This checks to see if we should add the organ donation vars? Shouldn't we say
``` 
if (dosomething_campaign_organ_donation_enabled)
  // add vars
  dosomething_campaign_add_organ_donation_vars($node);
```